tests/foreman/api/test_docker.py

```diff
@@ -1278,77 +1288,82 @@ def test_positive_create_using_cv(self):
         self.assertEqual(len(content_view.version), 1)
         cvv = content_view.read().version[0].read()
         promote(cvv, lce.id)
-        for compute_resource in (self.cr_internal, self.cr_external):
-            with self.subTest(compute_resource):
+
+        # publishing takes few seconds sometimes
+        retries = 10 if bz_bug_is_open(1452149) else 1
+        for i in range(retries):
```

Comment:
The act of asserting the resource availability is done as a part of creating the container. If it is not ready, an HTTP error is raised.
The following block is a simple `retry-until-limit` block. It is needed, since the http publishing takes a little longer than the CV-publishing task.

I'm not quite sure what you mean by "weird" on a simple `if` clause:
 ```
In [1]: retries = 2

In [2]: range(retries)
Out[2]: [0, 1]

In [3]: for i in range(retries):
   ...:     print(i == retries - 1)
   ...:     
False
True
```